action/library/README.md

```diff
@@ -315,16 +315,22 @@ Please note that not every call can be visible in `invocation log` entry.
 
 ### In-memory Cache Behaviour
 It wraps a simple action with cache. It caches a payload values added by a `doAction` action and 
-puts cached values in next invocations. It uses in-memory Guava cache implementation. The 
+puts cached values in next invocations.
+
+Cache implementation is selected the by `type` option. Knot.x provides OOTB `in-memory` cache factory using Guava cache.
```

Comment:
Was thinking about writing that legacy option is still available. Maybe no need to?